CIP-XXXX/README.md

```diff
@@ -0,0 +1,377 @@
+   conflict of interest in using custodial Ada for on-chain governance.
+2. The custodial Ada held by the administrator is on behalf of the entire
+   Cardano community. It would be impossible to fairly and accurately represent
+   the will of the entire community through any on- or off-chain governance
+   systems.
+
+<hr />
+
+#### Rule: Administrator Transparency
+
+##### Definition
+
+Budget administrators and custodians **MUST** be non-anonymous, legally
+recognized, public entities with a publicly disclosed board of directors and/or
+responsible parties.
+
+##### Rationale
+
+Individual budgets are likely to account for large amounts of value both in Ada
+and fiat computation. The responsible parties to manage and oversee the budget
+process should be publicly disclosed to bring confidence and accountability to
+the Cardano community.
+
+<hr />
+
+#### Rule: Radical Transparency
+
+##### Definition
+
+Budget allotments earmarked for specific purposes (buckets) **MUST** be kept
+separate and **MUST** be kept in secure, publicly disclosed addresses until
+disbursed as part of the normal budget process.
+
+##### Rationale
+
+If a single custodian is responsible for multiple facets of the budget (core
+development and research, for example) then it is imperative that the community
+be capable of auditing the funds distributed from the Treasury to the custodian
+and subsequently to contractor(s) engaging in work as part of the budget
+process. Co-mingling (pooling) funds from various buckets decreases transparency
+and should not be tolerated.
+
+<hr />
+
+#### Rule: Conflict Resolution
+
+##### Definition
+
+A budget proposal **MUST** define a jurisdiction and conflict resolution clause.
+Custodial funds **MUST NOT** be used for court or legal expenses by the
+administrator without an explicit _governance action_ authorization from the
+Cardano Community following the same criteria as budget approval.
+
+##### Rationale
+
+By defining a _forum selection clause_ within the scope of the
+community-approved budget we can reduce uncertainty for both the administrators
+and the community should a conflict or dispute arise.
+
+<hr />
+
+#### Rule: Administrator Fees
+
+##### Definition
+
+A budget administrator and custodian, in total, **MUST NOT** charge more than 5%
+of the total budget amount for administrative costs. All administrative costs
+**MUST** be defined as part of the budget proposal.
+
+##### Rationale
+
+It is imperative that the withdrawn Treasury funds be protected from exorbitant
+or malicious administrative costs that severely deplete the spendable funds. In
+alignment with the goal of community transparency, it's important that all
+administrative costs of budget management be disclosed as part of the budget
+process.
+
+<hr />
+
+#### Rule: Applicable Period
+
+##### Definition
+
+A budget proposal **MUST** specify its validity period. A budget proposal
+**MUST NOT** have a validity period longer than _net change period_ time.
```

Comment:
"Budget category" is confusing, it is better to use consistent term "bucket". In case there are many small budgets proposed within a year, maybe we can specify as follow

A singular entity SHALL NOT administer more than 50% of the Net Change Limit within the same Net Change Period.
A singular vendor SHALL NOT receive more than 50% of the Net Change Limit within the same Net Change Period
A singular vendor SHALL NOT receive more than 85% of the budget allocated to each bucket within the same Net Change Period.

Reply:
This is a really interesting one... I suppose a glossary of common terms may also be required here. IMO "bucket" is good for visualization purposes so I understand why it was chosen for the (current) budget process but it does not necessarily translate well (to non-English) in terms of what it actually is.